Neos.Neos/Classes/Service/Controller/DataSourceController.php

```diff
@@ -39,10 +47,9 @@ class DataSourceController extends AbstractServiceController
 
     /**
      * @param string $dataSourceIdentifier
-     * @param Node $node
      * @throws NeosException
      */
-    public function indexAction($dataSourceIdentifier, Node $node = null): void
+    public function indexAction($dataSourceIdentifier, string $node = null): void
```

Comment:
you removed the param annotation – I would instead suggest to keep it and add some docs instead – what is this `$node` string expected to contain?

Reply:
as per my main description:

> where previously Node $node was used to trigger property mapping (DataSourceController and WorkspaceController) we now pass the node as string and use the legacy node address factory to handle the legacy format manually. A followup will adjust everything to work with the new node address.

Reply:
OK, sorry, missed that one!